### PR TITLE
fixes some code problems.

### PR DESCRIPTION
**Pull request by Evankhell561**
before:
Screenshot_2

after:
image

### CODE REVIEW

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 1]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
 
 /datum/controller/processScheduler
 	// Processes known by the scheduler
-	var/tmp/datum/controller/process/list/processes = new
+	var/tmp/list/datum/controller/process/processes = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:6]: List don't support typized. Make var/tmp/list/processes

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 2]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Processes that are currently running
-	var/tmp/datum/controller/process/list/running = new
+	var/tmp/list/datum/controller/process/running = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:9]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 3]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Processes that are idle
-	var/tmp/datum/controller/process/list/idle = new
+	var/tmp/list/datum/controller/process/idle = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:12]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 4]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Processes that are queued to run
-	var/tmp/datum/controller/process/list/queued = new
+	var/tmp/list/datum/controller/process/queued = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:15]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 5]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Process name -> process object map
-	var/tmp/datum/controller/process/list/nameToProcessMap = new
+	var/tmp/list/datum/controller/process/nameToProcessMap = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:18]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 6]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Process last queued times (world time)
-	var/tmp/datum/controller/process/list/last_queued = new
+	var/tmp/list/datum/controller/process/last_queued = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:21]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 7]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Process last start times (real time)
-	var/tmp/datum/controller/process/list/last_start = new
+	var/tmp/list/datum/controller/process/last_start = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:24]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 8]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Process last run durations
-	var/tmp/datum/controller/process/list/last_run_time = new
+	var/tmp/list/datum/controller/process/last_run_time = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:27]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 9]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Per process list of the last 20 durations
-	var/tmp/datum/controller/process/list/last_twenty_run_times = new
+	var/tmp/list/datum/controller/process/last_twenty_run_times = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:30]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 10]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Process highest run time
-	var/tmp/datum/controller/process/list/highest_run_time = new
+	var/tmp/list/datum/controller/process/highest_run_time = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:33]: Too

File: code/controllers/ProcessScheduler/core/processScheduler.dm [hunk 11]
```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
...
 
 	// Process total run time
-	var/tmp/datum/controller/process/list/total_run_time = new
+	var/tmp/list/datum/controller/process/total_run_time = new
```
Comment (DeadLineSS13) [inline code/controllers/ProcessScheduler/core/processScheduler.dm:36]: Too

File: code/modules/client/preference_setup/laws/laws_pref.dm [hunk 12]
```diff
@@ -85,7 +85,7 @@
 		if(chosen_lawset)
 			var/path = valid_lawsets[chosen_lawset]
 			var/datum/ai_laws/lawset = new path()
-			var/datum/ai_law/list/laws = lawset.all_laws()
+			var/list/datum/ai_law/laws = lawset.all_laws()
```
Comment (DeadLineSS13) [inline code/modules/client/preference_setup/laws/laws_pref.dm:88]: Too

File: code/modules/hydroponics/spreading/spreading_growth.dm [hunk 13]
```diff
@@ -123,7 +123,7 @@
 	if(is_mature())
 		//Find a victim
 		if(!buckled_mob)
-			var/mob/living/list/targets = targets_in_range()
+			var/list/mob/living/targets = targets_in_range()
```
Comment (DeadLineSS13) [inline code/modules/hydroponics/spreading/spreading_growth.dm:126]: Too

File: code/modules/hydroponics/spreading/spreading_growth.dm [hunk 14]
```diff
@@ -241,7 +241,7 @@
 
 
 /obj/effect/vine/proc/targets_in_range()
-	var/mob/list/targets = list()
+	var/list/mob/targets = list()
```
Comment (DeadLineSS13) [inline code/modules/hydroponics/spreading/spreading_growth.dm:244]: Too

**Review by DeadLineSS13 — CHANGES_REQUESTED** [review]
I think that Nanako to agree with request changes

File: code/controllers/ProcessScheduler/core/processScheduler.dm (hunk 1 above)
Comment (Evankhell561) [reply code/controllers/ProcessScheduler/core/processScheduler.dm:6]: lists do support typized.

Comment (Evankhell561) [reply code/controllers/ProcessScheduler/core/processScheduler.dm:6]: > lists do support typized.

considering this, should I still make the changes you want me to make?

Comment (DeadLineSS13) [reply code/controllers/ProcessScheduler/core/processScheduler.dm:6]: > lists do support typized.

Try to get var type from list.

Comment (Evankhell561) [reply code/controllers/ProcessScheduler/core/processScheduler.dm:6]: > Try to get var type from list.

Lists do support typized. I wouldn't make that statement if I wasn't sure it was true, if you still want me to make the changes you are asking for I can make them.

File: code/modules/detectivework/forensics.dm [hunk 15]
```diff
@@ -12,9 +12,9 @@ proc/is_complete_print(var/print)
 	return stringpercent(print) <= FINGERPRINT_COMPLETE
 
 atom/var/list/suit_fibers
```
Comment (DeadLineSS13) [inline code/modules/detectivework/forensics.dm:14]: I think there you can use 
/atom
    var/list...
    var/...

File: code/modules/detectivework/forensics.dm [hunk 16]
```diff
@@ -11,10 +11,10 @@ var/const/FINGERPRINT_COMPLETE = 6
...
 	return stringpercent(print) <= FINGERPRINT_COMPLETE
 
-atom/var/list/suit_fibers
-atom/var/var/list/fingerprints
-atom/var/var/list/fingerprintshidden
-atom/var/var/fingerprintslast = null
+/atom/var/list/suit_fibers
+/atom/var/list/fingerprints
+/atom/var/list/fingerprintshidden
+/atom/var/fingerprintslast
```
Comment (DeadLineSS13) [inline code/modules/detectivework/forensics.dm]: Make 
/atom
   var/list/suit_fibers
   var/list/fingerprints
   var/list/fingerprintshidden
   var/fingerprintslast

File: code/controllers/ProcessScheduler/core/processScheduler.dm (hunk 1 above)
Comment (DeadLineSS13) [reply code/controllers/ProcessScheduler/core/processScheduler.dm:6]: > > Try to get var type from list.
> 
> 
> 
> Lists do support typized. I wouldn't make that statement if I wasn't sure it was true, if you still want me to make the changes you are asking for I can make them. 
> 
> 

Nanako and I have been talking about this. Nanako says it is not supported.

**Review by NanakoAC — APPROVED** [review]

**Pull request merged**